lib/README.md

```diff
@@ -16,6 +16,24 @@ To install globally type
 npm install -g odata-openapi
 ```
 
+## Local installation of the CLI

```

Comment:
npm link is not quite "local installation", see https://docs.npmjs.com/cli/v9/commands/npm-link

Reply:
adjusted